packages/salesforcedx-vscode-core/src/orgBrowser/orgMetadata.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import {
+  ContinueResponse
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as fs from 'fs';
+import * as path from 'path';
+import { Observable } from 'rxjs/Observable';
+import { isNullOrUndefined } from 'util';
+import * as vscode from 'vscode';
+import { channelService } from '../channels';
+import {
+  EmptyParametersGatherer,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from '../commands';
+import { nls } from '../messages';
+import { notificationService, ProgressNotification } from '../notifications';
+import { taskViewService } from '../statuses';
+import { getRootWorkspacePath, hasRootWorkspace, OrgAuthInfo } from '../util';
+
+export class ForceDescribeMetadataExecutor extends SfdxCommandletExecutor<string> {
+  private outputPath: string;
+
+  public constructor(outputPath: string) {
+    super();
+    this.outputPath = outputPath;
+  }
+
+  public build(data: {}): Command {
+    return new SfdxCommandBuilder()
+      .withDescription(
```

Comment:
@ruthemmanuelle Just wanted to check if this description was alright.

Reply:
Yeah, I think it's good. I like that it mirrors the CLI command's name (and the API call's). Shouldn't this be in a messages file, though?

Can you also update the list in this doc file? The commands are in alphabetical order by CLI command. https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/docs/articles/user-guide/cli-commands.md

Reply:
Okay sounds good! I just kept the messages in this file until we confirmed them, but I'll move them over to the messages file. This command actually won't be surfaced through the command palette, so would we still want to display it in that doc?

Reply:
This shouldn't be part of that list since the user will never use this through the command palette or any of the other ways we expose cli commands.

Reply:
Ah, ok. I didn't understand that. No need to add it to the doc if it won't be in the command palette. I think that adding it would confuse people.